keyboards/miiiw/blackio83/readme.md

```diff
@@ -0,0 +1,15 @@
+# MIIIW BlackIO83
+
+![BlackIO83](https://cdn.shopifycdn.net/s/files/1/0078/2863/5712/products/c34621677ffc53ea6d1c68ab82100749_1024x1024@2x.jpg?v=1612186093)
```

Comment:
Please rehost this on Imgur.  Shopify is known to remove/change urls and is not a good long term solution.

Reply:
Fixed.